### PR TITLE
Add support to "raw" credential by environment variable

### PR DESCRIPTION
**Pull request by rafaelhenrique**
#### What does this PR do?
Add support to GOOGLE_APPLICATION_CREDENTIALS and GOOGLE_SERVICE_ACCOUNT environment variables.

#### Why is it necessary?
The GOOGLE_APPLICATION_CREDENTIALS have filename (of a file in your disk) with Service Account credentials of Google Cloud Provider, the GOOGLE_SERVICE_ACCOUNT have the content of a common credentials file, in this case, you don't need store credentials file on your disk.

#### Examples:

To start a consumer you can do:
```
GOOGLE_APPLICATION_CREDENTIALS="credentials.json" python -m consumer
```

Or:
```
GOOGLE_SERVICE_ACCOUNT='{"type": "service_account", "project_id": "fake-project-123456", "private_key_id": "123"}' python -m consumer
```

PS: This Google Service Account is invalid, is only an example.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+### 1.1.2 (2021-10-20)
```
Comment (gilsondev) [inline CHANGES.md:1]: Changelog <3

File: tests/test_base.py [hunk 2]
```diff
@@ -58,9 +71,59 @@ def test_acknowledge_messages_call(mocked_subscriber_client, mocked_retry):
...
+        mocked_from_service_account_info.assert_called_once_with(
+            json.loads(google_service_account), audience=SUB_AUDIENCE
+        )
+        mocked_subscriber_client.assert_called_once_with(credentials=mocked_from_service_account_info())
+        assert subscriber_client == mocked_subscriber_client()
+
+
+@mock.patch("pydrinker_gcp.base.jwt.Credentials.from_service_account_info")
+@mock.patch("pydrinker_gcp.base.pubsub_v1.SubscriberClient")
+def test_get_subscriber_without_any_environment_variable(
```
Comment (iagorios) [inline tests/test_base.py:116]: this test does the same as `test_get_subscriber_with_service_account_value`. I believe in this one you want to check whether `ProviderError` is raised

Comment (rafaelhenrique) [reply tests/test_base.py:116]: Yes, this is wrong, i'll be fix that.

**Review by iagorios — APPROVED** [review]
LGTM

**Review by iagorios — APPROVED** [review]

**Pull request merged**